### PR TITLE
Add Holyiot nrf52840

### PR DESCRIPTION
**Pull request by Edrig**
if its possible to add hacktoberfest-accepted label to this ^^
thank

### CODE REVIEW

File: src/boards/holyiot_nrf52840/board.h [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+//--------------------------------------------------------------------+
+#define BLEDIS_MANUFACTURER  "Atelier-Du-Maker"
+#define BLEDIS_MODEL         "HOLY_NRF52840"
+
+//--------------------------------------------------------------------+
+// USB
+//--------------------------------------------------------------------+
+
+// Shared VID/PID with Feather nRF52840, will be disabled for building in the future
+#define USB_DESC_VID           0x239A
```
Comment (ladyada) [inline src/boards/holyiot_nrf52840/board.h:58]: hi you cannot use adafruit VID/PID for this non-adafruit product

**Review by ladyada — CHANGES_REQUESTED** [review]

**Comment by Edrig** [comment]
Hello, this is for use à  breakout for the Holyiot module, but available in open source [](). What VID / PID can we use in this case ?

**Comment by dhalbert** [comment]
You might first ask Holyiot. They already sell USB devices, and could allocate you one. Another alternative is https://github.com/pidcodes/pidcodes.github.com.

**Comment by Edrig** [comment]
hello im using the vid/pid 
that I was assigned following my request here :

https://github.com/adafruit/circuitpython/issues/2786

Thank you

**Comment by ladyada** [comment]
oh that was a long time ago! ok thanks!

**Review by ladyada — APPROVED** [review]

**Pull request merged**